src/FluentValidation.Tests/AbstractValidatorTester.cs

```diff
@@ -348,6 +348,15 @@ private class DerivedPerson : Person { }
 			Assert.Contains(testMessage, result.Errors.Select(failure => failure.ErrorMessage));
 		}
 
+		[Fact]
+		public void PropertyName_Dispays_Correctly_In_Messages_For_Properties_With_Periods() {

```

Comment:
There is a typo at `Displays`. Is this test necessary here as the functionality of `PropertyName` would be tested elsewhere already.